test/unit/dom.js

```diff
@@ -95,4 +95,46 @@ describe('DOM', function () {
       expect(helpers.fixture().querySelector('.my-circle').skated).to.equal(true);
     });
   });
+
+  describe('Document Fragments', function () {
+    var frag;
+    var created;
+    var attached;
+    var MyEl;
+
+    beforeEach(function () {
+      created = false;
+      attached = false;
+      MyEl = skate('my-element', {
```

Comment:
No they don't. That was prior to native support where we were able to unregister components.